web-registry/src/routes.tsx

```diff
@@ -1,23 +1,22 @@
-import React, { lazy } from 'react';
+import { lazy } from 'react';
```

Comment:
This file has been renamed to `routes.tsx` as it only contains routes configuration now.